### ISSUE TITLE
Big white square in GUI when deactivating toolbars

### ISSUE THREAD

**Issue by saogalde** [OpenHantek/openhantek#43]
Hi everyone,
I came up with the following bug, which is happening in Qt5.7.0 in OSX Yosemite. When I deactivate all toolbars in OpenHantek and re-open the program, a big white square appears. This doesn't happen  at the time when I deactivate all toolbars, but when I reopen the program with these settings. It can be easily solved by reactivating all toolbars again.

Normal screenshot
<img width="700" alt="without_problem" src="https://cloud.githubusercontent.com/assets/13224433/20463522/c59f444c-af13-11e6-94f6-a1e44ea81787.png">

Screenshot when error happening
<img width="700" alt="big_square" src="https://cloud.githubusercontent.com/assets/13224433/20463525/e82f9570-af13-11e6-8b48-048ddd1cc1a7.png">

# Console log
```
saogalde@macpro: ~/Downloads/openhantek-master/build/openhantek$ ./OpenHantek
Version 
QObject::connect: No such slot OpenHantekMainWindow::sendCommand()
QWidgetWindow(0x7ffe02f30ed0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36da0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36e00, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f30ed0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36da0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36e00, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36e00, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36da0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f30ed0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
QWidgetWindow(0x7ffe02f36da0, name="QToolBarClassWindow") ( QScreen(0x7ffe02d1e1c0, name="Color LCD") ): Attempt to set a screen on a child window.
```

**Comment by davidgraeff** [comment]
Related to #57 I guess

**Comment by davidgraeff** [comment]
Should be fixed now if Qt >= 5.4 is used for compiling
